bezier-rs/lib/src/lib.rs

```diff
@@ -453,6 +453,46 @@ impl Bezier {
 			.try_into()
 			.unwrap()
 	}
+
+	/// Returns a list of lists of points representing the hull points for all iterations at the point corresponding to `t` using de Casteljau's algorithm
+	/// the ith list represents the points of the ith iteration
+	/// https://pomax.github.io/bezierinfo/#decasteljau
+	pub fn hull(&self, t: f64) -> Vec<Vec<DVec2>> {
+		let mut current_points = Vec::new();
+		let mut hull_points = Vec::new();
+
+		match self.handles {
+			BezierHandles::Quadratic { handle } => {
+				current_points.push(self.start);
```

Comment:
You can move the push for `start` and `end` outside of the match statement (`start` in front, and `end` after).

Reply:
Ah, thanks for pointing that out ill change that